examples/hello-node-express-ts/index.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  Context,
+  context,
+  propagation,
+  trace,
+  Tracer,
+} from '@opentelemetry/api';
+import express, { Express, Request, Response } from 'express';
+import asyncHandler from 'express-async-handler';
+
+const app: Express = express();
+const hostname = '0.0.0.0';
+const port = 3000;
+
+app.get(
+  '/',
+  asyncHandler(async (_req: Request, res: Response) => {
```

Comment:
When eslint is enabled, there are lots of warnings around this function and how we handle the asynciness (?).  The asyncHandler does help us out on this. There may be a better way to do this... perhaps to be considered as a follow-up with a general reworking of the examples to more closely resemble our apps in other distros?

Reply:
I think the main question here is - what's the generally accepted way of "doing async" in express routes? I ended up doing this, but I'm not sure if it's considered "right" or not. It certainly helps satisfying eslint though.

Also...wondering if this is a "this is why fastify is in use" kinda moment

Reply:
Check out this [new commit](https://github.com/honeycombio/honeycomb-opentelemetry-node/pull/112/commits/20c47d174594d6153ad3d99765b58c7734d080d0).

Basically [it should be handling it fine](https://expressjs.com/en/guide/error-handling.html) - this asyncHandler function is eating the error and appeasing eslint, but it's excessive for what we're trying to do here. Just ignoring the eslint rule for now should suffice, and we can drop that dependency after all.

```js
app.get('/user/:id', async (req, res, next) => {
  const user = await getUserById(req.params.id)
  res.send(user)
})
```